src/app/auth/login/page.tsx

```diff
@@ -2,19 +2,28 @@
 
 import { Box } from '@primer/react'
 import { Login } from 'app/components/login/Login'
+import { useOrgsData } from 'hooks/useOrganizations'
 import { useSession } from 'next-auth/react'
 import { useRouter } from 'next/navigation'
 import { useEffect } from 'react'
 
 const LoginPage = () => {
   const session = useSession()
+  const orgsData = useOrgsData()
+
   const router = useRouter()
 
   useEffect(() => {
     if (session.data?.user) {
+      // redirect to org page if user has only one org
+      if (orgsData && orgsData.length === 1) {
```

Comment:
```suggestion
      if (orgsData?.length === 1) {
```

The nice thing about nullish coalescing is that we can usually drop the preceding `thing &&` check because it will equate to `undefined === 1` which is false :)